rootfs/scheduler/k8s.py

```diff
@@ -33,39 +33,106 @@
 }
 """
 
-RC_TEMPLATE = """\
+RCD_TEMPLATE = """\
 {
   "kind": "ReplicationController",
   "apiVersion": "$version",
   "metadata": {
     "name": "$name",
     "labels": {
-      "app": "$id",
-      "heritage": "deis"
+      "name": "$id"
     }
   },
   "spec": {
     "replicas": $num,
     "selector": {
-      "app": "$id",
+      "name": "$id",
       "version": "$appversion",
-      "type": "$type",
-      "heritage": "deis"
+      "type": "$type"
+    },
+    "template": {
+      "metadata": {
+        "labels": {
+          "name": "$id",
+          "version": "$appversion",
+          "type": "$type"
+        }
+      },
+      "spec": {
+        "containers": [
+          {
+            "name": "$containername",
+            "image": "$image",
+            "env": [
+            {
+                "name":"DEIS_APP",
+                "value":"$id"
+            },
+            {
+                "name":"DEIS_RELEASE",
+                "value":"$appversion"
+            }
+            ]
+          }
+        ]
+      }
+    }
+  }
+}
+"""
+
+RCB_TEMPLATE = """\
+{
+  "kind": "ReplicationController",
+  "apiVersion": "$version",
+  "metadata": {
+    "name": "$name",
+    "labels": {
+      "name": "$id"
+    }
+  },
+  "spec": {
+    "replicas": $num,
+    "selector": {
+      "name": "$id",
+      "version": "$appversion",
+      "type": "$type"
     },
     "template": {
       "metadata": {
         "labels": {
-          "app": "$id",
+          "name": "$id",
           "version": "$appversion",
-          "type": "$type",
-          "heritage": "deis"
+          "type": "$type"
         }
       },
       "spec": {
         "containers": [
           {
             "name": "$containername",
-            "image": "$image"
+            "image": "smothiki/slugrunner:4",
```

Comment:
Should be `quay.io/deisci/slugrunner:v2-alpha`, but evidently that job [is broken](https://travis-ci.org/deis/slugrunner/builds/95426679). I'll fix it.